### ISSUE TITLE
Implement GroupChat message handlers

### ISSUE THREAD

**Issue by oswaldoacauan** [bigbluebutton/bigbluebutton#4987]
To make HTML5 Client work on 2.1 we need to restructure the Chat collection and handle the new `GroupChat*` messages from redis.

In 2.0 each message was its own object in 2.1 each chat is an object with an array of messages. 

We already know that dealing with large arrays in Meteor/Mongo does not perform really well, so we will be separating it in two separate collections, one for the Chat and another for the Messages.

**Chat Document**
```
chatId: string
createdBy: object
name: string
access: enum
users: [string]
```
**Message Document**
```
id: string
chatId: string
correlationId: string
timestamp: int
sender: object
color: string
message: string
```

**Comment by oswaldoacauan** [comment]
@capilkey thats correct, thanks!

**Comment by oswaldoacauan** [comment]
@capilkey any idea why the `GroupChatCreatedEvtMsg` has a `correlationId`?

https://github.com/bigbluebutton/bigbluebutton/blob/master/bbb-common-message/src/main/scala/org/bigbluebutton/common2/msgs/GoupChatMsg.scala#L45

**Comment by capilkey** [comment]
I just looked at the usage in akka-apps and it isn't store anywhere. It looks like maybe it's used so the client could tell that the group it requested was actually created.

**Comment by antobinary** [comment]
Further changes implemented in #5929
